.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,34 @@
+---
+name: 💡 Feature Request
+about: I have a suggestion (and may want to implement it 🙂)!
```

Comment:
Might be worth asking about whether this is a request (for someone else to implement) or a proposal (to implement themselves)

Reply:
Added a dropdown for whether or not they would like to contribute (checkbox caused a 1/1 task complete on the issue and looks ugly + dropdown allows for a tentative maybe answer).